HtmlDiff/Utils.cs

```diff
@@ -9,7 +9,7 @@ public static class Utils
         private static Regex closingTagTexRegex = new Regex("^\\s*</[^>]+>\\s*$", RegexOptions.Compiled);
         private static Regex tagWordRegex = new Regex(@"<[^\s>]+", RegexOptions.Compiled);
         private static Regex whitespaceRegex = new Regex("^(\\s|&nbsp;)+$", RegexOptions.Compiled);
-        private static Regex wordRegex = new Regex(@"[\w\#@]+", RegexOptions.Compiled | RegexOptions.ECMAScript);
+        private static Regex wordRegex = new Regex(@"[\w\#@]+", RegexOptions.ECMAScript);
```

Comment:
.NET Core has a bug and throws a runtime error with these lines OR'd.  Had to remove RegexOptions.Compiled for the time being